apps/sv/frontend/src/components/ValidatorOnboardingSecrets.tsx

```diff
@@ -57,58 +76,152 @@ const ValidatorOnboardingSecrets: React.FC = () => {
         </Typography>
       )}
 
+      <Stack direction="column" mb={4} spacing={1}>
+        <Typography variant="h6">Party Hint</Typography>
+        <TextField
+          error={partyHint === '' || !VALID_PARTY_ID_REGEX.test(partyHint)}
+          autoComplete="off"
+          id="create-party-hint"
+          placeholder="<organization>-<function>-<enumerator>"
+          inputProps={{ 'data-testid': 'create-party-hint' }}
+          onChange={e => setPartyHint(e.target.value)}
+          value={partyHint}
+        />
+      </Stack>
+
       <DisableConditionally
-        conditions={[{ disabled: prepareOnboardingMutation.isPending, reason: 'Loading...' }]}
+        conditions={[
+          { disabled: prepareOnboardingMutation.isPending, reason: 'Loading...' },
+          { disabled: partyHint === '', reason: 'No Party Hint' },
+          {
+            disabled: !VALID_PARTY_ID_REGEX.test(partyHint),
+            reason: 'Party Hint is invalid',
+            severity: 'warning',
+          },
```

Comment:
Do we need both conditions for partyHint? I think the second one is adequate. The empty string will fail the regex check